app/view_objects/find/results_view.rb

```diff
@@ -0,0 +1,260 @@
+module Find
+  # FIND:TODO Will be fleshed out in an upcoming ticket
+  class ResultsView
+    attr_reader :query_parameters
+
+    include ActionView::Helpers::NumberHelper
+
+    MAXIMUM_NUMBER_OF_SUBJECTS = 43
```

Comment:
Is this used?

Reply:
It's used by another method in this class. Have only moved methods used directly in the results page at this point in time though.